docs/README.md

```diff
@@ -0,0 +1,47 @@
+# Testing github pages locally
+
+This includes steps to test local changes related to github pages.
+
+Note: Current repo github pages leverages  `jekyll-theme-minimal` theme.
+
+* Install dependencies
+
+```bash
+sudo apt update
+sudo apt install ruby-full build-essential zlib1g-dev
+```
+
+* Create Gemfile under `docs` directory
+
+```bash
+cat <<EOF > Gemfile
+source "https://rubygems.org"
```

Comment:
Any reason you aren’t including this file in the repo?

Reply:
Didn't want it to interfere with what the github workflow autogenerates. 
My guess is we get version upgrades.. etc for free with the workflow and would be good to keep it as is.